apiary.apib

```diff
@@ -0,0 +1,66 @@
+FORMAT: 1A
+# Rails API Target
+
+# Group Users
+
+## Create User [api/v1/users/] [POST]
+
++ Request Bad
+
+    + Headers
+
+            Content-Type: application/json
+
+    + Body
+
+            {
+              "email":"user@example.com",
```

Comment:
Look at this https://apiblueprint.org/documentation/advanced-tutorial.html#data-structures